monitors/CRMonitor.js

```diff
@@ -27,14 +31,23 @@ class CRMonitor {
   }
 
   // Queries all monitored wallet ballance for collateralization ratio against a given threshold.
-  checkWalletCrRatio = async priceFunction => {
+  checkWalletCrRatio = async () => {
     // yield the price feed at the current time.
-    const contractTime = this.empClient.getLastUpdateTime();
-    const priceFeed = priceFunction(contractTime);
+    const price = this.priceFeed.getCurrentPrice();
+
+    if (!price) {
+      this.logger.warn({
+        at: "CRMonitor",
+        message: "Cannot compute wallet collateralization ratio because price feed returned invalid value",
+        price
```

Comment:
nit: is there a reason this is not key-valued like:
```
price: price
```

Reply:
You can write it like that if the current variable name is the same as the key you want to create. Happy to change if you have a stylistic preference to not use that sort of syntax.

Reply:
ok cool. We should do this in all places then.